api/openapi-spec/swagger.json

```diff
@@ -4266,8 +4266,7 @@
           "$ref": "#/definitions/io.k8s.api.core.v1.LocalObjectReference"
         },
         "name": {
-          "type": "string",
```

Comment:
What happened with all these changes?

Reply:
this was fixed after rerunning the openapi codegen script with the updated binary